server/helpers.go

```diff
@@ -44,10 +44,10 @@ func getListOfThemes() ([]byte, error) {
 
 // ApplyTheme function takes a theme name and installs it in current terminal
 func applyTheme(theme string) ([]byte, error) {
-	log.Println("Applying the theme: " + theme)
+	terminal := os.Getenv("TERMINAL")
```

Comment:
@ash-ishh this assumes that the **_TERMINAL_** env will always exists, but that's not true. The original approach as well as this approach fails when user is using the non-default terminal.